fwutil/lib.py

```diff
@@ -210,7 +210,7 @@ class SquashFs(object):
     OVERLAY_MOUNTPOINT_TEMPLATE = "/tmp/image-{}-overlay"
 
     def __init__(self):
-        image_stem = self.next_image.lstrip(self.OS_PREFIX)
+        image_stem = self.next_image.replace(self.OS_PREFIX, EMPTY, 1)
```

Comment:
What is the leading non-whitespace character that can be present which needs to be replaced? Just curious. Maybe a comment would be helpful to explain why we need to use `replace`.

Reply:
@jleveque The `replace` is needed because we hit the similar situation: https://bugs.python.org/issue33362

Reply:
I see. Yes, `lstrip` is wrong here. However, while this will most likely never happen, `replace` will replace the first instance of the substring even if it occurs in the middle of the string. A more proper solution would be the following:

```python
        image_stem = self.next_image
        if image_stem.startswith(self.OS_PREFIX):
            image_stem = image_stem[len(self.OS_PREFIX):]
```

Reply:
@jleveque done